docs/glossary.md

```diff
@@ -74,38 +74,51 @@ In Sematic there is no actual concept of pipeline. We have no abstraction to
 describe pipelines.
 
 What we call a "pipeline" is typically the root Sematic Function. It is the
-Sematic Function on which you called `.resolve()`.
+Sematic Function on which you called `runner.run(some_sematic_function())`.
 
 See [Root function](#root-entry-point-function).
 
-## Resolvers
+## Runners
 
-Resolvers dictate how your pipeline gets "resolved". Resolving a pipeline means
+{% hint style="info" %}
+This concept used to be referred to as Resolvers. So don't
+worry if you're familiar with that terminology! Everything
+you know about Resolvers applies to Runners as well, except
+that `.resolve(...)` has been renamed to `.run(...)`. 
+{% endhint %}
+
+Runners dictate how your pipeline gets "ran". Running a pipeline means
```

Comment:
```suggestion
Runners dictate how your pipeline get executed. Running a pipeline means
```

Reply:
Changed to "run" as I believe it is more grammatically valid. But I think the idea here is to maintain parallelism between the words used for "Runner" and the action they perform. That parallelism was present in the original "Resolver"/"gets resolved" structure of these docs, and I am trying to preserve it here.